schemas/mw-article-delete.js

```diff
@@ -0,0 +1,54 @@
+{
+    "title": "MediaWiki Article Delete",
+    "description": "Represents a MW Article Delete Edit event",
+    "type": "object",
+    "properties": {
+        // global event fields
+        "uri": {
+            "type": "string",
+            "format": "uri",
+            "description": "the unique URI identifying the event"
+        },
+        "request_id": {
+            "type": "string",
+            "pattern": "^[a-fA-F0-9]{8}(-[a-fA-F0-9]{4}){3}-[a-fA-F0-9]{12}$",
+            "description": "the unique UUID v1 ID of the event derived from the X-Request-Id header"
+        },
+        "event_id": {
+            "type": "string",
+            "pattern": "^[a-fA-F0-9]{8}(-[a-fA-F0-9]{4}){3}-[a-fA-F0-9]{12}$",
+            "description": "the unique ID of this event; added by the event bus"
+        },
+        "dt": {
```

Comment:
We could also consider moving to a nested `event` object, as sketched in https://phabricator.wikimedia.org/T116247#1757410. This would simplify schema reuse somewhat.